website/content/pro/copilot/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: OpenBB Copilot
+sidebar_position: 7
+description: Learn how to use OpenBB Copilot to interact with the OpenBB Terminal Pro
+keywords:
+- OpenBB Copilot
+- copilot
+- voice command
+- agent
+- assistant
+- Natural language processing
+- Large language model
+- OpenAI
+---
+
+import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
+import ReactPlayer from 'react-player'
+
+
+<HeadTitle title="Copilot | OpenBB Terminal Pro Docs" />
+
+OpenBB copilot is your companion to interact with the OpenBB Terminal Pro.
+
+## Getting started
+
+<ReactPlayer width="50%" height="50%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/6466c3e3-111b-4246-ae9b-b6403712c862' />
+
+To open OpenBB Copilot, click on the icon located at the bottom right of the
+Terminal Pro screen.
+
+You can interact with OpenBB Copilot either through typing text or using your
+voice.  Under most circumstances, you can treat OpenBB Copilot like your very
+own personal research assistant. OpenBB Copilot can answer general financial
+questions, use the data displayed in the dashboard to perform analytical tasks,
+and even interact with files that you choose to upload.
+
+We'll be exploring each of these features the following sections below.
+
+## Understanding chat history
+
+<ReactPlayer width="50%" height="100%" playing loop muted='true' volume='0' url='https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/312510fb-fb17-474c-8698-a2960134e285' />
+
+OpenBB Copilot is a conversational agent. This means that OpenBB Copilot uses
+previous messages and answers in the current chat to help answer your query.
+This allows you to guide OpenBB Copilot to perform the task you want, or ask
+follow-up questions.
+
+In the example above, the user first asked about the Price-to-earnings (P/E)
```

Comment:
capitzlization on Price-to-earnings

Reply:
> Capitalization of sub-headers, "Understanding Chat History"

I'm following the pattern set in the Terminal Pro section of the docs, which doesn't use Title Case for subheadings.